packages/ember-cookies/.gitignore

```diff
@@ -19,6 +17,6 @@ yarn-error.log
 testem.log
 .DS_Store
 
-# ember-try
-.node_modules.ember-try/
-package.json.ember-try
+# .md copies from release
```

Comment:
to make those files explorable on github, we keep them in the root folder.

to make those files packed at publish time, we copy them to the addon folder.
This allows to keep single source of truth and at the same time keep all the nice parts of DX